app/components/UserNavigation/index.jsx

```diff
@@ -77,13 +77,12 @@ const UserNavigation = ({
     (path === routes.profileView(account) ||
       path === routes.userCommunities(account));
 
-  const isGlobalModerator = useMemo(
+  const isGlobalAdmin = useMemo(
```

Comment:
Do we still have integer properties with keys to check moderators? Do not we have a list of roles?